packages/gatsby-plugin-google-gtag/src/__tests__/gatsby-ssr.js

```diff
@@ -15,8 +16,31 @@ it(`does not crash when no pluginConfig is provided`, () => {
 const DO_NOT_TRACK_STRING = `!(navigator.doNotTrack == "1" || window.doNotTrack == "1")`
 
 describe(`respectDNT`, () => {
+  it(`adds a preconnect link for Google Analytics`, () => {
```

Comment:
could you move this test out of respectDNT describe? We're missing an root describe here but for now moving the `it` statement to the root is fine.

Reply:
Done!